src/bot.ts

```diff
@@ -478,7 +482,7 @@ export class DiscordBot {
     }
 
     // Update presence because sometimes discord misses people.
-    this.userSync.OnUpdateUser(msg.author).then(() => {
+    await this.userSync.OnUpdateUser(msg.author).then(() => {
```

Comment:
Yes, we could. The only reasons I didn't is because
1) when I originally wrote the PR it wasn't and I just resolved the merge conflicts and check that it still works.
2) it'll change all the indents, thus increasing the diff and potential for merge conflicts
If you'd prefer, I'll collapse them. I won't be able to do that this week though.